src/main/features/claim/views/reason.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import csrfProtection, emailInput, saveAndContinueButton, errorSummary, textArea %}
 
-{% set heading = t('Why you’re owed the money') %}
+{% set heading = t('Briefly explain your claim') %}
```

Comment:
There's still a few places that need updating after this change:

```
$ rg "Why you’re owed the money"
src/test/features/claim/routes/reason.ts
33:        .expect(res => expect(res).to.be.successful.withText('Why you’re owed the money'))
43:        .expect(res => expect(res).to.be.successful.withText('Why you’re owed the money'))
62:          .expect(res => expect(res).to.be.successful.withText('Why you’re owed the money', 'div class="error-summary"'))

src/main/features/claim/views/check-and-send.njk
110:    {{ row('Why you’re owed the money:', draftClaim.reason.reason, paths.reasonPage.uri, bold = true ) }}
```
Should it be changed on check and send @RonanFitzgerald ?